src/content/docs/en/guides/cms/preprcms.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Prepr CMS & Astro
+description: Add content to your Astro project using Prepr as a CMS
+type: cms
+service: Prepr CMS
+stub: true
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Prepr.io](https://www.prepr.io/), is a headless CMS for building websites with Segment targeting.
```

Comment:
Should this be capitalised or lower case?
```suggestion
[Prepr.io](https://www.prepr.io/), is a headless CMS for building websites with segment targeting.
```

Reply:
@TheOtterlord  I think it's fine with the caps, since we use it as a feature name. But i'm fine too if you prefer lower case.

Reply:
Hey! I'd actually be more permissive here and let you explain what that is, because it sounds like an interesting (factual) distinction, but I can't find "Segment Targeting" in your docs.  (There's no search, and it's not one of the main pages to click, so I'm not exactly sure where to get more info on this.)

So I'm happy to approve and merge this as is, but I'd also be OK with something like:

 > is a headless CMS for building websites with segment targeting for delivering personalized content.
 
I don't know if that is exactly accurate, but I'd be OK with a sentence of that style, if you prefer! I think that would be more informative for people reading and not knowing your features? But it's up to you!  Let us know, or make a change, and as long as it's just factual like that, I'm sure it will be fine!